src/main/scripts/update_versions.py

```diff
@@ -0,0 +1,75 @@
+import logging
+
+import requests
+import yaml
+
+logging.basicConfig(level=logging.INFO, format="%(levelname).1s %(message)s")
+
+products = ["bitbucket", "jira", "bamboo", "confluence", "crowd"]
+suffix = "-jdk11"
+current_lts = {"bitbucket": "7.17", "jira": "8.20", "confluence": "7.13"}
+
+
+def update_chart(product, new_version):
+    chart_file = f'../charts/{product}/Chart.yaml'
+    with open(chart_file, "r") as stream:
+        content = stream.read()
+        doc = yaml.safe_load(content)
+        current_version = doc['appVersion']
+        logging.info("current version: %s", current_version)
+
+    new_content = content.replace(current_version, f"{new_version}{suffix}")
```

Comment:
Naively I have used `yaml.dump()` first but that just re-shuffle the whole file.